basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -44,11 +55,35 @@ static boolean containsDuplicates( String[] args){
     }
 
 
-//Calculating Averages: Write a method that accepts an array of integers and calculates and
-// returns the average of all the values in the array.
+    //Calculating Averages: Write a method that accepts an array of integers and calculates and
+    // returns the average of all the values in the array.
+    public static int calculatingAverage(int[] num){
+        int result = 0;
+        int len = num.length;
+        for (int i = 0; i < num.length; i++){
+            result += num[i];
+        }
+        return result/len;
+    }
 
-// Arrays of Arrays: Given an array of arrays calculate the average value for each array
-// and return the array with the lowest average.
+    // Arrays of Arrays: Given an array of arrays calculate the average value for each array
+    // and return the array with the lowest average.
+    public static int arrayAverage(int[][] multiArr){
+        int max = Integer.MAX_VALUE;
```

Comment:
`max` should be renamed to something like `minimum` since it holds the lowest average.